Yoast/wordpress-seo
# Created issue when zip size is too big

## Summary


This PR can be summarized in the following changelog entry:

* N/A - Creates an issue during the RC process when the zip is too large.

## Relevant technical choices:

* Also added `EDITOR` env to change the editor used to edit changelogs.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have a `GITHUB_ACCESS_TOKEN` and a `SLACK_DEV_PLUGIN_CHANNEL_TOKEN` token set in your environment variables or in your `.env` file.
* Have an artifact.zip file that is bigger that 5MB (you can simply rename a big file), in the root folder of `wordpress-seo`. 
* Set the `API_BASE` variable in `verify-zip-size.js` to a custom repo (you can simply create a new one for this).
* Create two milestones: `release/1.0` and `hotfix/1.0.1`.
* Run `grunt verify-zip-size --plugin-version=1.0`.
* Run `grunt verify-zip-size --plugin-version=1.0.1`.
* Make sure the issues are created and the slack notifications are sent.

## UI changes

## Documentation

## Quality assurance


Fixes #14059
